analyze-documents-ai-docintelligence/manifest.json

```diff
@@ -1,52 +1,32 @@
 {
     "title": "Document Intelligence: Analyze complex documents using Document Intelligence and Azure OpenAI",
     "description": "This workflow uses Forms Recognizer connector to covert complex documents into markdown and Azure Open AI connector to analyze the image content, and provides response to specific questions, based on that image.",
-    "prerequisites": "To use this template, you need to meet the following requirements: \n- Configure an Azure Open AI instance and an embedding model. For more information, see [Create a chat with your data logic app project](https://github.com/Azure/logicapps/tree/master/LogicApps-AI-RAG-Demo). \n - [Create an Azure AI service or Document Intelligence resource](https://go.microsoft.com/fwlink/?linkid=2302318). You then use the keys and endpoint to create a connection.",
-    "detailsDescription": "",
-    "tags": [
-        "RAG-Document-Ingestion"
-    ],
+    "artifacts": [],
     "skus": [
         "standard"
     ],
-    "kinds": [
-        "stateful",
-        "stateless"
-    ],
-    "details": {
-        "By": "Microsoft",
-        "Type": "Workflow",
-        "Trigger": "Request",
-        "Category": "AI,RAG"
-    },
-    "artifacts": [
-        {
-            "type": "workflow",
-            "file": "workflow.json"
+    "workflows": {
+        "analyze-documents-ai-docintelligence": {
+            "name": "analyze_Documents_Ai_Docintelligence"
```

Comment:
Just think everywhere when it is only one workflow we are calling them default. How do we update the code there, this contract we already set with partner team that when there is only one workflow we are calling them default. Once it is published you can't change it to accelerator.